packages/parser/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@solid-primitives/parser",
+  "version": "0.0.101",
+  "description": "A primitive to tokenize your solid-components to enable custom parsing.",
+  "author": "Vincent Van Dijck <vandijckv@gmail.com>",
+  "contributors": [],
+  "license": "MIT",
+  "homepage": "https://github.com/solidjs-community/solid-primitives/tree/main/packages/parser#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/solidjs-community/solid-primitives.git"
+  },
+  "bugs": {
+    "url": "https://github.com/solidjs-community/solid-primitives/issues"
+  },
+  "primitive": {
+    "name": "parser",
+    "stage": 0,
+    "list": [
+      "createPagination"
```

Comment:
Please add the exported items to the list.

Reply:
done!